backend/dataall/modules/notebooks/services/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+Contains the code needed for service layer.
+The service layer is a layer where all business logic is aggregated
+"""
+
+from dataall.db.api import TargetType
+from dataall.modules.notebooks.services import services, permissions
+from dataall.modules.common.sagemaker import permissions as common_permission
+
+__all__ = ["services", "permissions", "common_permission"]
+
+TargetType("notebook", permissions.GET_NOTEBOOK, permissions.UPDATE_NOTEBOOK)
```

Comment:
It will be important in the future that we document very well what a module needs to have. For example I see new joiners forgetting to add a TargetType

Reply:
But this is a note to ourselves, implementation-wise I think it is what it should be

Reply:
Yes! Perhaps we can also come up with better-defined API for importing that will make you to pass it

Reply:
Moved it to `NotebookApiInterface` and documented in the code